src/PKSim.Core/Services/TransportContainerUpdater.cs

```diff
@@ -58,9 +59,27 @@ public void SetDefaultSettingsForTransporter(ISimulationSubject simulationSubjec
          foreach (var transporterContainer in simulationSubject.AllMoleculeContainersFor<TransporterExpressionContainer>(transporter))
          {
             updateTransporterContainerFromTemplate(transporterContainer, null, transportType);
+            updateFractionExpressedEpithelial(transporterContainer, transportType);
          }
       }
 
+      private void updateFractionExpressedEpithelial(TransporterExpressionContainer transporterContainer, TransportType transportType)
+      {
+         if (!transportType.IsOneOf(TransportType.Efflux, TransportType.Influx))
```

Comment:
@Yuri05 Only udating the behavior for Influx and Efflux. Not sure what BiDirectional or pgp should do

Reply:
PgP: same as Efflux I would say.
Bidirectional: no idea. Maybe @PavelBal can answer.

Reply:
ok updating for pGP and leaving as is for BiDirectional